src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-bucket-form/rgw-bucket-form.component.ts

```diff
@@ -407,6 +413,21 @@ export class RgwBucketFormComponent extends CdForm implements OnInit, AfterViewC
     }
   }
 
+  updateBucketRateLimit() {
+    // Check if bucket ratelimit has been modified.
+    const ratelimitvalue: RgwRateLimitConfig = this.rateLimitComponent.getRateLimitFormValue(); // TODO change any here;
```

Comment:
the usual casing we follow in angular is camelCase
```suggestion
    const rateLimitConfig: RgwRateLimitConfig = this.rateLimitComponent.getRateLimitFormValue(); // TODO change any here;
```

Reply:
Updated